packages/messaging/test/testing-utils/make-fake-firebase-services.ts

```diff
@@ -34,12 +48,23 @@ export function makeFakeApp(options: FirebaseOptions = {}): FirebaseApp {
     automaticDataCollectionEnabled: true,
     delete: async () => {},
     messaging: null as any,
-    installations() {
-      return {
-        getId: () => Promise.resolve('FID'),
-        getToken: () => Promise.resolve('authToken'),
-        delete: () => Promise.resolve()
-      };
-    }
+    installations: null as any
+  };
+}
+
+export function makeFakeInstallations(): FirebaseInstallations {
+  return {
+    getId: () => Promise.resolve('FID'),
```

Comment:
Very much a nit, but you can do `getId: async () => 'FID',`. Also the other ones.

Reply:
Nice! Changed here and other places where we return a promise.